main.go

```diff
@@ -139,12 +139,26 @@ func getConnWithRetry(flags *flagStorage) (c *gcsx.Connection, err error) {
 ////////////////////////////////////////////////////////////////////////
 
 // Mount the file system according to arguments in the supplied context.
+
+func getUserAgent(AppName string) string {
```

Comment:
AppName should be in camelCase ? i.e. appName

Reply:
fixed